### PR TITLE
Add value getter

### LINKED ISSUE #56: Add value getter/setter to all Derivables for easier bindings
The functions `#get()` and `#set()` will be aliased as `get value()` and `set value()` to facilitate easier access and specifically bindings (for example binding to `<input>` elements using `ngModel` in Angular).

### PR DESCRIPTION
**Pull request by njirem**
closes #56

### CODE REVIEW

File: src/derivable/atom.spec.ts [hunk 1]
```diff
@@ -40,6 +40,27 @@ describe('derivable/atom', () => {
         });
     });
 
+    describe('#value', () => {
+        it('should call #get() when getting the #value property', () => {
+            const a$ = atom('a');
+            const s = spy(a$, 'get');
+
+            // Use the getter
+            a$.value;
```
Comment (WouterSpaak) [inline src/derivable/atom.spec.ts]: Just for speccing clarity, I'd like to see a test wether `a$.value` equals 'a'.

File: src/derivable/atom.ts [hunk 2]
```diff
@@ -49,13 +48,19 @@ export class Atom<V> extends Derivable<V> {
...
+        const oldValue = this._value;
         if (!equals(newValue, oldValue)) {
-            this.value = newValue;
+            this._value = newValue;
             processChangedAtom(this, oldValue, this.version++);
         }
     }
 
+    /**
+     * JavaScript getter/setter access to the `#get` and `#set` methods
```
Comment (WouterSpaak) [inline src/derivable/atom.ts]: This jsdoc seems a bit too concise. I'd quite like to explicitly see that setting `#value` calls `#set()` and getting `value` calls `#get()`.

File: src/derivable/atom.spec.ts [hunk 3]
```diff
@@ -40,6 +40,27 @@ describe('derivable/atom', () => {
         });
     });
 
+    describe('#value', () => {
```
Comment (pavadeli) [inline src/derivable/atom.spec.ts]: This test can be moved to `derivable.spec.ts` behind an `if (isAtom(value$))` guard. This way it is also applied to Lenses.

File: src/derivable/lens.spec.ts [hunk 4]
```diff
@@ -79,6 +79,29 @@ describe('derivable/lens', () => {
         });
     });
 
+    describe('#value', () => {
```
Comment (pavadeli) [inline src/derivable/lens.spec.ts]: This can be removed when the Atom tests are moved to `derivable.spec.ts`.

File: src/derivable/derivable.ts [hunk 5]
```diff
@@ -28,6 +28,13 @@ export abstract class Derivable<V> implements TrackedObservable {
      */
     abstract get(): V;
 
+    /**
+     * `#value` is an alias for the `#get()` method on the Derivable.
+     * Getting `#value` will call `#get()` and return the value.
+     * `#value` is readonly on a Derivable
```
Comment (pavadeli) [inline src/derivable/derivable.ts]: Derivable => derivation

**Pull request merged**